README.rst

```diff
@@ -360,23 +360,28 @@ example:
 
 .. code:: python
 
+   import os
+
    # using it in argparse
    parser.add_argument('current_host', type=str, default=os.environ['SM_CURRENT_HOST'])
 
    # using it as variable
    current_host = os.environ['SM_CURRENT_HOST']
 
+.. _header-n1032:
+
 SM_HOSTS
 ~~~~~~~~
 
-.. code:: json
+.. code:: shell
 
    SM_HOSTS='["algo-1","algo-2"]'
 
 JSON encoded list containing all the hosts . Usage example:
 
 .. code:: python
 
+   import os
    import json
 
    # using it in argparse
```

Comment:
the type can be either `str` or `json.loads` - not sure that you need `nargs`

Reply:
Changing type to `str`